solution.txt

```diff
@@ -205,11 +192,10 @@ Exercise 15:
 // Complete Exercise 15 below...
 
 let total = 0;
-
-numArrays.flat(3).forEach(bigSum);
-
-function bigSum(n) {
-  total += n;
-}
+numArrays.forEach(function(numArr) {
+  numArr.forEach(function(num) {
+    total += num;
+  });
+});
```

Comment:
nice! indentation looks great throughout your document, so thanks for making your code readable 😄